doc/contributor/updating-ruby.md

```diff
@@ -14,3 +14,13 @@ questions.
 
 * Update version information in `RubyLanguage`
 * Update `doc/user/compatibility.md`
+
+## Updating bundled gems
+The current list of bundled gems their versions are found here:
+https://github.com/ruby/ruby/blob/ruby_2_3/gems/bundled_gems
+
+To update a bundled gem, follow these steps:
+1. Remove the current gem and gemspec from `lib/gems/2.3.0/gems` and `lib/gems/2.3.0/specifications`
+2. Run the gem install command with the desired version. E.g. `./bin/ruby -rbundler-workarounds -S gem install rake -v 10.4.2 --no-doc`
+3. Update the project `.gitignore` to allow the newly install gem sources and gemspec
+4. If the gem installs any executables like `rake`. Add these to the `.gitignore` if not already and verify there is no shebang line and remove executable permissions on the file.
```

Comment:
The list formatting did not work out (use the `rich diff` button above). Maybe an extra space before `1.`.
An not-numbered list might be more consistent with above.